pandas/io/parsers/readers.py

```diff
@@ -131,6 +132,10 @@
     parsing time and lower memory usage.
 squeeze : bool, default False
     If the parsed data only contains one column then return a Series.
+
+    .. deprecated:: 1.4.0
+        Append ``.squeeze("columns")`` to the call to ``read_csv`` to squeeze
```

Comment:
```suggestion
        Append ``.squeeze("columns")`` to the call to ``{func_name}`` to squeeze
```

This docstring is for both `read_csv` and `read_table`